solution/ui/regulations/eregs-vite/src/components/resources/ResourceExportBtn.vue

```diff
@@ -1,12 +1,21 @@
 <template>
-    <div class="resource_btn_container">
-        <button class="default-btn action-btn search_resource_btn desktop-btn-label`" @click="createCSV">
-            <span class="desktop-btn-label">Download Spreadsheet (CSV)</span><span class="mobile-btn-label">CSV</span>&nbsp;&nbsp;
-            <svg width="18" height="17" viewBox="0 0 18 17" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <path
-                    d="M1.78361 16.5C1.36371 16.5 0.996298 16.35 0.681372 16.05C0.366447 15.75 0.208984 15.4 0.208984 15V11.425H1.78361V15H15.4304V11.425H17.005V15C17.005 15.4 16.8475 15.75 16.5326 16.05C16.2177 16.35 15.8503 16.5 15.4304 16.5H1.78361ZM8.60699 12.675L3.54194 7.85L4.67043 6.775L7.81968 9.775V0.5H9.39431V9.775L12.5436 6.775L13.672 7.85L8.60699 12.675Z"
-                    fill="white" />
-            </svg>
+    <div class="resource_btn_container" style="padding-left:5px;">
+        <button id="export-csv" class="default-btn action-btn search_resource_btn desktop-btn-label`"
+            @click="createCSV">
+            <span v-if="!downloadingCSV">
+                <span class="desktop-btn-label">
+                    Download Spreadsheet (CSV)
+                </span>
+                <span class="mobile-btn-label">CSV</span>
+                <svg width="18" height="17" viewBox="0 0 18 17" fill="none" xmlns="http://www.w3.org/2000/svg"
+                    v-if="!downloadingCSV">
+                    <path
+                        d="M1.78361 16.5C1.36371 16.5 0.996298 16.35 0.681372 16.05C0.366447 15.75 0.208984 15.4 0.208984 15V11.425H1.78361V15H15.4304V11.425H17.005V15C17.005 15.4 16.8475 15.75 16.5326 16.05C16.2177 16.35 15.8503 16.5 15.4304 16.5H1.78361ZM8.60699 12.675L3.54194 7.85L4.67043 6.775L7.81968 9.775V0.5H9.39431V9.775L12.5436 6.775L13.672 7.85L8.60699 12.675Z"
+                        fill="white" />
+                </svg></span>
+            <span class="desktop-btn-label" style="width:100%" v-else>
+                <simple-spinner spinnerType="button" :size="'small'"></simple-spinner>
```

Comment:
the colon (`:`) in front of an attribute is shorthand for `v-bind`, which binds a prop from the component context to the attribute.  In our case we don't need to bind anything to `size` because we're explicitly setting it as a string.  So this component should look like this:

`<simple-spinner spinnerType="button" size="small"></simple-spinner>`